api/src/main/java/io/strimzi/api/kafka/model/KafkaExporterSpec.java

```diff
@@ -0,0 +1,174 @@
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Representation of the Kafka Exporter deployment.
+ */
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+        "replicas", "image", "groupRegex",
+        "topicRegex", "watchedNamespace",
+        "resources", "logging",
+        "enableSarama", "template"})
+@EqualsAndHashCode
+public class KafkaExporterSpec implements UnknownPropertyPreserving, Serializable {
+
+    private static final long serialVersionUID = 1L;
+    public static final int DEFAULT_REPLICAS = 1;
+
+    private int replicas;
+    private String image;
+    private String groupRegex;
+    private String topicRegex;
+
+    private String watchedNamespace;
+    private ResourceRequirements resources;
+    private Probe livenessProbe;
+    private Probe readinessProbe;
+    private String logging;
+    private boolean enableSarama;
+    private KafkaExporterTemplate template;
+    private Map<String, Object> additionalProperties = new HashMap<>(0);
+
+    @Description("The namespace the Exporter Operator should watch.")
+    public String getWatchedNamespace() {
+        return watchedNamespace;
+    }
+
+    public void setWatchedNamespace(String watchedNamespace) {
+        this.watchedNamespace = watchedNamespace;
+    }
+
+    @Description("The number of pods in the `Deployment`.")
+    @Minimum(1)
+    public int getReplicas() {
+        return replicas;
+    }
+
+    public void setReplicas(int replicas) {
+        this.replicas = replicas;
+    }
+
+    @Description("The docker image for the pods.")
+    @JsonInclude(JsonInclude.Include.NON_DEFAULT)
+    public String getImage() {
+        return image;
+    }
+
+    public void setImage(String image) {
+        this.image = image;
+    }
+
+    @Description("Regex that determines which consumer groups to collect.")
+    @JsonProperty(required = true)
+    public String getGroupRegex() {
+        return groupRegex;
+    }
+
+    public void setGroupRegex(String groupRegex) {
+        this.groupRegex = groupRegex;
+    }
+
+    @Description("Regex that determines which topics to collect.")
+    @JsonProperty(required = true)
+    public String getTopicRegex() {
+        return topicRegex;
+    }
+
+    public void setTopicRegex(String topicRegex) {
+        this.topicRegex = topicRegex;
+    }
+
+    @Description("Turn on Sarama logging.")
+    @JsonInclude(value = JsonInclude.Include.NON_NULL)
+    public boolean getEnableSarama() {
+        return enableSarama;
+    }
+
+    public void setEnableSarama(boolean enableSarama) {
+        this.enableSarama = enableSarama;
+    }
+
+    @Description("Only log messages with the given severity or above. Valid levels: [debug, info, warn, error, fatal].")
+    @JsonInclude(value = JsonInclude.Include.NON_NULL)
```

Comment:
You seem to use sometimes

```
@JsonInclude(JsonInclude.Include.NON_NULL)
```

and sometimes

```
@JsonInclude(value = JsonInclude.Include.NON_NULL)
```

Could we use just one in a single file? Or do they have different meanings?